src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,48 @@
+package step1;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set 크기 확인")
+    void size() {
+        int size = numbers.size();
+        assertThat(size).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1,2,3})
+    @DisplayName("Set 값 존재 확인")
+    void contains(int num) {
+        assertThat(numbers.contains(num)).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1:true", "2:true", "3:true", "4:false", "5:false"}, delimiter = ':')
+    @DisplayName("결과값이 다른 경우 확인")
+    void containsAllCase(int input, String expected) {
+        String result = String.valueOf(numbers.contains(input));
```

Comment:
`expected` 네이밍을 활용해주셨군요 👍 
```suggestion
        String actual = String.valueOf(numbers.contains(input));
```

테스트 코드에서 
실제값 : actual
예상값: expected

이렇게 표현하는것도 테스트 코드를 단순화 시키고 가독성에 도움이 된다고 생각합니다 :)